### PR TITLE
add xflx and coingecko for flx

### PR DESCRIPTION
**Pull request by opptylabs**

### CODE REVIEW

**Comment by olivbau** [comment]
I dont really know how flowx platform works, but xflx and flx price seems to be different
image

File: src/tokens/sui.json [hunk 1]
```diff
@@ -254,7 +254,21 @@
...
+  },
+  {
+    "chainId": 1,
+    "address": "0x65ed6d4e666fcbc1afcd9d4b1d6d4af7def3eeeeaa663f5bebae8101112290f6::xflx::XFLX",
+    "decimals": 8,
+    "name": "XFlowX",
+    "symbol": "XFLX",
+    "logoURI": "https://raw.githubusercontent.com/sonarwatch/token-lists/main/images/sui/xflx.png",
+    "extensions": {
+      "coingeckoId": "flowx-finance"
```
Comment (olivbau) [inline src/tokens/sui.json]: XFlowX  price is not FlowX price

**Comment by opptylabs** [comment]
Well, it's no equal if you swap immediatly, but you can convert to FLX at a ratio of 1:1 if you wait 90 days.

[xFLX Governance Token Documentation](https://docs.flowx.finance/tokenomics/xflx-governance-token)

Should we use immediate swapable value or real locked value ?

**Comment by olivbau** [comment]
It's best not to put coingecko id on the xflx. The xflx can still be detected by SW if it is present in SW-supported pools.

**Pull request merged**